### PR TITLE
Update dashboard document for Kubernetes environment

### PR DESCRIPTION
**Pull request by dulanjalidilmi**
## Purpose
Update docs for https://github.com/wso2/micro-integrator/issues/2135

### CODE REVIEW

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 1]
```diff
@@ -66,6 +66,25 @@ Follow the steps given below to configure the MI servers to publish data to the
     group_id = "mi_dev"
     node_id = "dev_node_2"
     ```
+    
+    If MI server is deployed in a Kubernetes environment, add following configuration too to the `deployment.toml`.
```
Comment (samgnaniah) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:70]:
```suggestion
    If the Micro Integrator server is deployed in a Kubernetes environment, add the following configuration to the `deployment.toml` file. 
```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 2]
```diff
@@ -66,6 +66,25 @@ Follow the steps given below to configure the MI servers to publish data to the
     group_id = "mi_dev"
     node_id = "dev_node_2"
     ```
+    
+    If MI server is deployed in a Kubernetes environment, add following configuration too to the `deployment.toml`. 
+    
+    Note: This capability is released as a product update on todo. If you do not have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/#!) now.
+    
+    Limitation: Currently the dashboard does not support deployments with replicas
```
Comment (samgnaniah) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:74]:
```suggestion
    Limitation: Currently the dashboard does not support deployments with replicas.
```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 3]
```diff
@@ -66,6 +66,25 @@ Follow the steps given below to configure the MI servers to publish data to the
...
+
+    If communication via Ingress:
```
Comment (samgnaniah) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:76]:
```suggestion
    If communicating via Ingress, use the following configuration:
```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 4]
```diff
@@ -66,6 +66,25 @@ Follow the steps given below to configure the MI servers to publish data to the
...
+    ```toml
+    dashboard_url = "https://{hostname/ip}:{port}/dashboard/api/"
+    management_hostname = "<INGRESS_HOSTNAME>"
+    ```
+    
+    If communication via Service
```
Comment (samgnaniah) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:82]:
```suggestion
    If communicating via Service instead, use the following configuration:
```

**Review by Mariangela — CHANGES_REQUESTED** [review]
DO NOT MERGE ATM

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**